tests/cli/test_revpath.py

```diff
@@ -125,6 +126,36 @@ def test_can_parse(self):
             self.assertTrue(isinstance(dataset, IDataset))
             self.assertEqual(None, project)
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    @scoped
+    def test_ambiguous_format(self):
+        test_dir = scope_add(TestDir())
+
+        # this dataset can be loaded as either imagenet or image_dir
```

Comment:
I don't think it is a blocker now, but I'm not sure if this should be true. `ImageDir` should be a fallback format, when no others match. Maybe we can have a json file instead (eg. with `annotation` and `annotations` fields), or add a couple of test formats in this test?

Reply:
> ImageDir should be a fallback format, used when no others match.

I'm not sure I agree, but we can discuss that later. For now, I've changed the test to use a dataset that is _definitely_ ambiguous.